src/description.js

```diff
@@ -0,0 +1,50 @@
+'use strict';
+
+var fs = require('fs');
+var path = require('path');
+var s = require('util').format;
+var marked = require('marked');
```

Comment:
Why I did not tested for file existence. Which `fs.readFile` is already doing actually.

Node.js: "fs.exists() will be deprecated"
io.js: "fs.exists() is deprecated"

We need a way to test whether it's a String or a file prior to hit the filesystem IMO. (even dumb)